src/main/java/org/opensearch/ad/indices/ADIndex.java

```diff
@@ -34,12 +35,11 @@ public enum ADIndex {
         false,
         ThrowingSupplierWrapper.throwingSupplierWrapper(AnomalyDetectionIndices::getAnomalyDetectorMappings)
     ),
-    // @anomaly-detection.create-detector Commented this code until we have support of Job Scheduler for extensibility
-    // JOB(
-    // AnomalyDetectorJob.ANOMALY_DETECTOR_JOB_INDEX,
-    // false,
-    // ThrowingSupplierWrapper.throwingSupplierWrapper(AnomalyDetectionIndices::getAnomalyDetectorJobMappings)
-    // ),
+    JOB(
+        AnomalyDetectorJob.ANOMALY_DETECTOR_JOB_INDEX,
+        false,
+        ThrowingSupplierWrapper.throwingSupplierWrapper(AnomalyDetectionIndices::getAnomalyDetectorJobMappings)
+    ),
```

Comment:
No complaint here, just pointing to the diff above as a reason we shouldn't be using `//` comments on blocks of code :)